src/membership/members.rs

```diff
@@ -122,14 +150,16 @@ decl_storage! {
         /// MemberId to assign to next member that is added to the registry
         pub NextMemberId get(next_member_id) build(|config: &GenesisConfig<T>| config.first_member_id): T::MemberId = T::MemberId::from(DEFAULT_FIRST_MEMBER_ID);
 
-        /// Mapping of member ids to their corresponding primary accountid
+        /// Mapping of member ids to their corresponding primary account_id
         pub AccountIdByMemberId get(account_id_by_member_id) : map T::MemberId => T::AccountId;
 
-        /// Mapping of members' account ids to their member id.
+        /// Mapping of members' primary account ids to their member id.
```

Comment:
Can we also drop all the optional keys here in storage now? was just a bad workaround for the benefit of Pioneer. Probably better in separate PR though.

Reply:
It will require changes in pioneer obviously.
Keeping it as separate PR is a good idea.